### PR TITLE
Dev notification

### PR DESCRIPTION
**Pull request by viv9k**
### Functionality:
Added Notification Feature

### Solution:
1. Fixed Header UI.
2. Header Icons fixed.
3. Added Feature to handle notifications like add and remove.
4. Added feature to handle active notification.
5. Added UI with proper scroll for notification view.
6. Added Important link to notification actions.

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
Check notification icon on top right.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -89,10 +89,16 @@ import { DataTableComponent } from './body/common-tools/data-table/data-table.co
...
+import { MatIconModule } from '@angular/material/icon';
+import { MatBadgeModule } from '@angular/material/badge';
 import { TeamsCardComponent } from './body/view-organization-details/teams-card/teams-card.component';
 import { OrgDocumentsComponent } from './body/view-organization-details/org-documents/org-documents.component';
 import { Patch6Component } from './patches/patch6/patch6.component';
+<<<<<<< HEAD
+import { NotificationCenterComponent } from './header/notification-center/notification-center.component';
+=======
 import { CopyrightComponent } from './body/copyright/copyright.component';
+>>>>>>> cf747c82ccd827c20f023722781de295b5b8a47d
```
Comment (Pendyala-thanvi) [inline src/app/app.module.ts:101]: Accept incoming changes, please.

Comment (viv9k) [reply src/app/app.module.ts]: resolved

**Review by Pendyala-thanvi — APPROVED** [review]
Ship It

**Review by Pendyala-thanvi — COMMENTED** [review]
Resolve merge conflicts

**Pull request merged**